tests/cypress/support/commands.ts

```diff
@@ -85,12 +85,19 @@ Cypress.Commands.add('addFleetGitRepo', ({ repoName, repoUrl, branch, path, gitA
 });
 
 // 3 dots menu selection
-Cypress.Commands.add('open3dotsMenu', (name, selection) => {
+Cypress.Commands.add('open3dotsMenu', (name, selection, checkMenu=false) => {
   // Open 3 dots button
   cy.contains('tr.main-row', name).within(() => {
     cy.get('.icon.icon-actions', { timeout: 5000 }).click();
   });
-  if (selection) {
+  if (checkMenu === true) {
+    cy.get('.list-unstyled.menu > li').each(($el, index, $list) => {
+      if ($el.text() != selection) {
+        cy.log(`Cannot move cluster to the another workspace because "${selection}" is not present. Current Menu: ${$el.text()}`);
```

Comment:
Log is nice but no assertion is placed, hence it will always work.
Need to find one type `cy.contains(selection).should('not.exist')`
I will try to find one on my end that works always

Reply:
This is the best I could came up with:

```ts
Cypress.Commands.add('open3dotsMenu', (name, selection, checkNotInMenu=false) => {
  // Open 3 dots button
  cy.contains('tr.main-row', name).within(() => {
    cy.get('.icon.icon-actions', { timeout: 5000 }).click();
  });

  if (checkNotInMenu === true) {
    cy.get('.list-unstyled.menu > li').each(($el) => {
        if ($el.text() != selection) {
        cy.log(`Cannot do action with specified value "${selection}" since it is not present. Current Menu value is: "${$el.text()}"`);
        cy.get('ul.list-unstyled.menu').contains(selection).should('not.exist')
      }        
    });
  }
```

Please notice I changed `checkMenu` to `chckNotInMenu` given that the check is done negatively. If accepted it needs to be updated in `e2e.ts` as well